README.md

```diff
@@ -16,18 +16,21 @@ A very fast input-validation and transformer library.
 
 ## Description
 
-Ever need the performance of very fast input-validation libraries such as [ts-json-validator](https://www.npmjs.com/package/ts-json-validator) with the powerful API of [zod](https://www.npmjs.com/package/zod)? ShapeShift got you covered!
+Ever need the performance of very fast input-validation libraries such as [`ts-json-validator`] with the powerful API of [`zod`]? ShapeShift got you covered!
```

Comment:
We should remove the mention to ts-json-validator, it's extremely fast in comparison to this library.

Reply:
@vladfrangu I'll let you say yay or nay to this

Reply:
Honestly we should probably just drop the mention of either libraries and just describe that we're one of the fastest object parsing libraries (excluding JSON ones)

Reply:
Bump @favna

Reply:
done